guava/src/com/google/common/base/CharMatcher.java

```diff
@@ -596,7 +596,6 @@ private static boolean isSmall(int totalCharacters, int tableLength) {
   /**
    * Sets bits in {@code table} matched by this matcher.
    */
-  @GwtIncompatible // java.util.BitSet
```

Comment:
FYI, I will probably leave these as `@GwtIncompatible`, since they're used only from other `@GwtIncompatible` code. (We might as well spare the GWT compiler the trouble of compiling and discarding them, and we might as well give ourselves the option of adding non-GWT-compatible code to them in the future.) I'll update the comment, though.